### PR TITLE
Add bug report and feature request templates

### PR DESCRIPTION
**Pull request by melink14**
These are the default templates github recommended with some small edits for rikaikun.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**System (please complete the following information):**
```
Comment (Stephie) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]:
```suggestion
**System details (OS, browser version, rikaikun version)**
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ - OS: [e.g. windows 10, catalina]
+ - Browser Version: [e.g. 68]
+ - Rikaikun Version: [e.g. 0.10.1]
```
Comment (Stephie) [inline .github/ISSUE_TEMPLATE/bug_report.md:29]: is rikaikun lowecase or capitalized?
```suggestion
 - rikaikun Version: [e.g. 0.10.1]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**System (please complete the following information):**
+ - OS: [e.g. windows 10, catalina]
```
Comment (Stephie) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]:
```suggestion
 - OS: [e.g. Windows 10, macOS Catalina]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+ - Browser Version: [e.g. 68]
```
Comment (Stephie) [inline .github/ISSUE_TEMPLATE/bug_report.md:28]:
```suggestion
 - Browser Version: [e.g. Chrome 84]
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (melink14) [reply .github/ISSUE_TEMPLATE/bug_report.md:29]: lower!

**Pull request merged**